inst/qml/Descriptives.qml

```diff
@@ -247,6 +247,7 @@ Form
 
 			CheckBox { name: "standardErrorMean";	label: qsTr("S. E. mean")							}
 			CheckBox { name: "standardDeviation";	label: qsTr("Std.deviation");		checked: true	}
+			CheckBox { name: "cOfVariation";	    label: qsTr("Coefficient of Variation");			}
```

Comment:
very nitpicky, but in QML files we use tabs rather than spaces:
```suggestion
			CheckBox { name: "cOfVariation";		label: qsTr("Coefficient of Variation");			}
```
Rstudio is terrible at using tabs and I'd recommend using a different editor for that (e.g., notpade++ or vscode)

Reply:
I used kate on linux, an editor that recognizes Qt. I will try next time to do better. Thanks very much for the detailed hints!

Reply:
kate should be fine for this!